### PR TITLE
add python solution for 1131

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-1131.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+id: usaco-1131
+source: USACO Bronze 2021 US Open
+title: Acowdemia I
+author: Juheon Rhee
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob1_bronze_open21.html)
```
Comment (ryanchou-dev) [inline solutions/usaco-1131.mdx:8]:
```suggestion
[Official Analysis (Java)](http://www.usaco.org/current/data/sol_prob1_bronze_open21.html)
```

File: solutions/usaco-1131.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+while citations[length - 1] < length and length >= 0:
+        length -= 1
+
+cur_h, length = length, len(citations)
+
+if cur_h is not n:
+        for i in range(cur_h, -1, -1):
+                if i <= cur_h - l:
+                        break
+                citations[i] += 1
```
Comment (ryanchou-dev) [inline solutions/usaco-1131.mdx:29]: indent with tabs

**Review by ryanchou-dev — CHANGES_REQUESTED** [review]

File: solutions/usaco-1131.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+id: usaco-1131
+source: USACO Bronze 2021 US Open
+title: Acowdemia I
+author: Juheon Rhee
+---
+
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_prob1_bronze_open21.html)
+
+## Implementation
+
```
Comment (ryanchou-dev) [inline solutions/usaco-1131.mdx:11]: (optional)

could you add the time complexity?

Comment (jjliewie) [reply solutions/usaco-1131.mdx:11]: yep

File: solutions/usaco-1131.mdx (hunk 2 above)
Comment (jjliewie) [reply solutions/usaco-1131.mdx:31]: is it not?

Comment (ryanchou-dev) [reply solutions/usaco-1131.mdx:31]: nope

Comment (ryanchou-dev) [reply solutions/usaco-1131.mdx:31]: bump

**Review by ryanchou-dev — CHANGES_REQUESTED** [review]
indent with tabs!!

**Review by ryanchou-dev — APPROVED** [review]
err, ill just do it for you

gj! ⭐

File: solutions/usaco-1131.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+```py
+n, l = map(int, input().split())
+citations = sorted([*map(int, input().split())], reverse=True)
+length = len(citations)
+
+while citations[length - 1] < length and length >= 0:
+	length -= 1
+
+cur_h, length = length, len(citations)
```
Comment (SansPapyrus683) [inline solutions/usaco-1131.mdx:25]: separate lines

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-1131.mdx (hunk 4 above)
Comment (jjliewie) [reply solutions/usaco-1131.mdx:25]: i cant unless i use a temp variable

Comment (SansPapyrus683) [reply solutions/usaco-1131.mdx:25]: fair enough

**Comment by SansPapyrus683** [comment]
OOF CONFLICTS

**Comment by ryanchou-dev** [comment]
@jjliewie ya, so resolve the conflicts and i think it's good to merge

**Comment by jjliewie** [comment]
> @jjliewie ya, so resolve the conflicts and i think it's good to merge

wait what conflicts

**Review by nwatx — APPROVED** [review]

**Comment by ryanchou-dev** [comment]
> wait what conflicts? they're all resolved?

image

**Comment by SansPapyrus683** [comment]
Ryan look at the bottom

File: solutions/usaco-1131.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+<PySection>
+
+```py
+n, l = map(int, input().split())
+citations = sorted([*map(int, input().split())], reverse=True)
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1131.mdx:19]:
```suggestion
citations = sorted(list(map(int, input().split())), reverse=True)
```

**Review by Aadit-Ambadkar — APPROVED** [review]
just this and that's it!

**Comment by Aadit-Ambadkar** [comment]
image
never seen that before...

File: content/extraProblems.json [hunk 6]
```diff
@@ -1038,6 +1038,19 @@
...
+      "uniqueId": "usaco-1131",
+      "name": "Acowdemia I",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1131",
+      "source": "Bronze",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Binary Search"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1131"
```
Comment (SansPapyrus683) [inline content/extraProblems.json:1052]:
```suggestion
        "kind": "internal"
```

**Pull request merged**